src/components/ActionButtonBar.vue

```diff
@@ -12,11 +12,19 @@
           </button>
         </li>
         <li>
+          <label
+            for="fileChoose"
+            class="btn btn-default"
+            title="Used To Upload Previous Completed or Semi-Completed AIA Forms"
```

Comment:
Title needs translation

Reply:
I noticed if this is on mobile, title will not be apparent since it needs a hover on the button to display the title therefore less accessibility for user. 
Instead I displayed the information on the blue box above to include "You can also upload previous completed or semi-completed AIA forms as JSON file." Same thing with French version.